### PR TITLE
Activity cursor pagination

### PR DESCRIPTION
**Pull request by IlgizGilmanov**
# Summary

#### A brief description of the pull request.

Pagination was added to activity page

#### Screenshots in case of UI changes

Снимок экрана 2020-10-13 в 19 10 38
Снимок экрана 2020-10-13 в 19 10 56
Снимок экрана 2020-10-13 в 19 11 47

# Test plan

#### List of steps to manually test introduced functionality:

- Login
- Go to [Activity page](http://localhost:8080/activity)

# Review notes

#### While reviewing pull-request (especially when it's your pull-request), please make sure that:

- you named PR close to issue name and linked that issue
- you understand what problem is solved by PR and how is it solved
- new tests are in place, no redundant tests
- required ENV variables added and described in `.env.example` and added to Heroku
- associated Heroku review app works correctly with introduced changes

### CODE REVIEW

File: components/atoms/Loader/Loader.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { transparentize } from 'polished';
 
 const Wrapper = styled.div(
   ({ theme: { colors } }) => css`
-    position: absolute;
+    position: fixed;
```
Comment (nadezhdakharchuk) [inline components/atoms/Loader/Loader.js:7]: Why do we need to fix the position?

File: components/atoms/PaginationButton/PaginationButton.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    },
+    disabled,
+    customStyles,
+  }) => css`
+    display: flex;
+    padding: 0.5rem;
+    background-color: transparent;
+    border: 2px solid ${grey};
+
+    ${disabled &&
```
Comment (nadezhdakharchuk) [inline components/atoms/PaginationButton/PaginationButton.js:17]: Maybe it should be better to use the ternary operator here?

File: public/images/icons/left-pointer.svg [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```
Comment (nadezhdakharchuk) [inline public/images/icons/left-pointer.svg:2]: Please minify svg

**Review by nadezhdakharchuk — APPROVED** [review]
Please fix linter warnings or tests

**Review by Regina-Gimazova — APPROVED** [review]

**Pull request merged**